src/test/java/com/oracle/nosql/spring/data/test/app/CustomerRepository.java

```diff
@@ -68,12 +68,15 @@
     List<Customer> queryByLastNameOrderByFirstNameDesc(String lastname);
     List<Customer> getByLastNameOrderByFirstNameAsc(String lastname);
 
-    // not implemented
-//    // Enabling ignoring case for an individual property
-//    List<Customer> findByLastNameIgnoreCase(String lastname);
-//    // Enabling ignoring case for all suitable properties
-//    List<Customer> findByLastNameAndFirstNameAllIgnoreCase(String lastname,
-//        String firstname);
+    // Enabling ignoring case for an individual property
+    List<Customer> findByLastNameIgnoreCase(String lastname);
+    // Enabling ignoring case for all suitable properties
+    List<Customer> findByLastNameAndFirstNameAllIgnoreCase(String lastname,
+        String firstname);
+    // Enable ignore case only for firstName field
```

Comment:
What's the difference here? I would have expected that findByLastNameAndFirstNameIgnoreCase() would ignore case on both first and last name - is that not true? Why the two variants?

Reply:
For findByLastNameAndFirstNameIgnoreCase IgnoreCase appliens only to FirstName. If user wants to apply to all fields it has to say AllIgnoreCase at the end.